scadalts-ui/src/components/amcharts/LineChart.js

```diff
@@ -0,0 +1,103 @@
+import * as am4core from '@amcharts/amcharts4/core';
+import * as am4charts from '@amcharts/amcharts4/charts';
+import am4themes_animated from '@amcharts/amcharts4/themes/animated';
+import httpClient from 'axios';
+import BaseChart from './BaseChart';
+
+am4core.useTheme(am4themes_animated);
+export class LineChart extends BaseChart {
+	constructor(chartReference, color, domain = '.') {
+		super(chartReference, 'XYChart', color, domain);
+	}
+	displayControls(scrollbarX, scrollbarY, legend) {
+		if (scrollbarX !== undefined && scrollbarX == 'true') {
```

Comment:
use ?? operator to clear undefined and null into other value in a more readable way